site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst

```diff
@@ -25,36 +25,117 @@ The following desktop environments support integrated account syncing, including
+
+  .. figure:: /_static/images/nextcloud/nextcloud-linux-desktop-1.png
+    :width: 60%
+    :alt: nextcloud login
+
+2. From your server's Nextcloud Service page, go to **Interfaces** and copy the LAN address.
+
+  .. figure:: /_static/images/nextcloud/nextcloud-mac-step3-lan.png
+    :width: 60%
+    :alt: nextcloud interfaces
+
+3. Enter your LAN address under **Server Address** and click **Next**.
+
+  .. figure:: /_static/images/nextcloud/nextcloud-linux-desktop-2.png
+    :width: 60%
+    :alt: nextcloud server path
+
+4. This will launch a page in your web browser, click **Log In** and then **Grant access** to link the desktop client. You can close this browser window afterwards.
+
+  .. figure:: /_static/images/nextcloud/nextcloud-mac-step5.png
+    :width: 60%
+    :alt: nextcloud login and grant access
+
+5. Next, configure the local directory that you want to sync with Nextcloud. You may use the default or change it, and edit the sync settings to desired. When satisfied, click **Connect**.
+
+  .. figure:: /_static/images/nextcloud/nextcloud-linux-desktop-3.png
+    :width: 60%
+    :alt: nextcloud add account
+
+6. Files will begin to sync immediately and you will see a green check when this is complete.
+
+  .. figure:: /_static/images/nextcloud/nextcloud-linux-desktop-5.png
+    :width: 60%
+    :alt: nextcloud sync
+
+7.  That's it! From this desktop client you will recieve notifications, control accounts and syncing, and quickly access your Apps' WebUI pages
 
 Tor Setup
 .........
 You will first need to have the :ref:`Tor daemon running<tor-linux>`.
 
-1. On your desktop application. Click the account in the top left -> Settings, then in Settings, click Network, then "Specify proxy manually as" and "SOCKS5 proxy." Enter "127.0.0.1" for the Host and "9050" for the port.
-2. Close the Settings screen and click the account in the top left again, then "Add Account."
-3. On the following screen, click "Log in," then enter your Nextcloud Tor server address, which you can copy from the Nextcloud page on your Embassy -> Interfaces -> Tor. This must start with `http://` and end with `.onion`. Click Next.
+1. On your desktop application. Click the account in the top left > Settings, then click Network. Choose **Specify proxy manually as** and **SOCKS5 proxy**. Enter **127.0.0.1** for the Host and **9050** for the port.
+
+  .. figure:: /_static/images/nextcloud/nextcloud-linux-tor-1.png
+    :width: 60%
+    :alt: nextcloud network settings
+
+2. Close the Settings screen and click the account in the top left again, then **Add Account**.
+
+  .. figure:: /_static/images/nextcloud/nextcloud-linux-tor-2.png
+    :width: 60%
+    :alt: nextcloud add account
+
+3. On the following screen, click **Log in your Nextcloud**, then enter your Nextcloud Tor server address, which you can copy from the Nextcloud page on your **startOS > Interfaces > Tor**. This must start with **`http://`** and end with **`.onion`**. Click **Next**.
```

Comment:
Did you test this with https by chance?  This is preferred if it works

Reply:
Will test it and correct.